test/unit/cli-domain-package-static-files.js

```diff
@@ -6,69 +6,75 @@ var path = require('path');
 var sinon = require('sinon');
 var _ = require('underscore');
 
-var packageStaticFiles,
-    error,
-    mocks;
-
-var initialise = function(mocks, params, cb){
-  packageStaticFiles = injectr('../../src/cli/domain/package-static-files.js', mocks, { console: console });
-  packageStaticFiles(params, function(e, r){
-    error = e;
-    cb();
-  });
-};
-
-var cleanup = function(){
-  error = null;
-  mocks = {
-    'babel-core': {
-      transform: sinon.stub().returns({
-        code: 'this-is-transpiled'
-      })
-    },
-    'clean-css': sinon.stub().returns({
-      minify: function(){
-        return { styles: 'this-is-minified'};
-      }
-    }),
-    'fs-extra': {
-      copySync: sinon.spy(),
-      ensureDirSync: sinon.spy(),
-      existsSync: sinon.stub().returns(true),
-      lstatSync: sinon.stub().returns({
-        isDirectory: function(){ return true; }
-      }),
-      readFileSync: sinon.stub().returns('some content'),
-      writeFileSync: sinon.spy()
-    },
-    'node-dir': { paths: sinon.stub().yields(null, { files:[] })},
-    path: {
-      basename: path.basename,
-      dirname: function(){
-        return path.dirname.apply(this, _.toArray(arguments)).replace(/\\/g, '/');
-      },
-      extname: path.extname,
-      join: function(){
-        return path.join.apply(this, _.toArray(arguments)).replace(/\\/g, '/');
+describe('cli : domain : package-static-files', function(){
```

Comment:
Agree. That's why I am starting to take one piece off (minify part). Definitely gonna tackle other little pieces but I would do in separate PRs if you don't mind